src/main/java/org/antipathy/scalafmtmvn/FormatMojo.java

```diff
@@ -28,6 +28,10 @@
     private boolean skip;
     @Parameter(defaultValue = "${project}", required = true, readonly = true)
     private MavenProject project;
+    @Parameter(defaultValue = "false", required = true, readonly = true)
+    private boolean skipTestSources;
+    @Parameter(defaultValue = "false", required = true, readonly = true)
```

Comment:
Could you add a blurb to the readme around setting these properties/effect if not set?